stdlib/queue.gr

```diff
@@ -137,6 +138,37 @@ provide let pop = queue => {
   }
 }
 
+/**
+ * Converts a queue into a list of its elements.
+ *
+ * @param queue: The queue to convert
+ * @returns A list containing all queue values
+ *
+ * @since v0.6.0
+ */
+provide let toList = queue => {
+  let lst = List.init(size(queue), i => match (queue.array[queue.headIndex +
+    i]) {
+    Some(n) => n,
+    None => fail "Impossible",
```

Comment:
Needs a better impossible message

Reply:
How about `"Impossible: Empty Bucket In Queue.toList`?

Reply:
Isn't it more about trying to access a bucket that doesn't exist?

Reply:
fair, so how about: `Impossible: Attempted To Access Non Existant Bucket In Queue.toList`

Reply:
```suggestion
    None => fail "Impossible:  Attempted to access non-existent bucket in Queue.toList",
```
Sounds good to me, fixed spelling and capitalization in suggestion